lms/static/js/bookmarks/views/bookmarks_list.js

```diff
@@ -0,0 +1,124 @@
+;(function (define, undefined) {
+    'use strict';
+    define(['gettext', 'jquery', 'underscore', 'backbone', 'moment'],
+        function (gettext, $, _, Backbone, _moment) {
+
+        var moment = _moment || window.moment;
+
+        return Backbone.View.extend({
+
+            el: '#courseware-results-list',
+            coursewareContentElement: '#course-content',
+
+            errorIcon: '<i class="fa fa-fw fa-exclamation-triangle message-error" aria-hidden="true"></i>',
+            loadingIcon: '<i class="fa fa-fw fa-spinner fa-pulse message-in-progress" aria-hidden="true"></i>',
+
+            errorMessage: gettext('An error has occurred. Please try again.'),
+            loadingMessage: gettext('Loading'),
+
+            url: '/api/bookmarks/v0/bookmarks/',
+
+            events : {
+                'click .bookmarks-results-list-item': 'visitBookmark'
+            },
+
+            initialize: function (options) {
+                this.template = _.template($('#bookmarks_list-tpl').text());
+                this.loadingMessageView = options.loadingMessageView;
+                this.errorMessageView = options.errorMessageView;
+                this.courseId = $(this.el).data('courseId');
+                this.langCode = $(this.el).data('langCode');
+                _.bindAll(this, 'render', 'breadcrumbTrail', 'userFriendlyDate', 'bookmarkUrl');
+            },
+
+            render: function () {
+                var data = {
+                    bookmarks: this.collection.models,
+                    breadcrumbTrail: this.breadcrumbTrail,
+                    userFriendlyDate: this.userFriendlyDate,
+                    bookmarkUrl: this.bookmarkUrl
+                };
+                this.$el.html(this.template(data));
+                this.delegateEvents();
+                return this;
+            },
+
+            loadBookmarks: function () {
```

Comment:
can we rename this to something like `fetchAndShowBookmarksList`  or `showBookmarksList` ?